src/library_browser.js

```diff
@@ -225,6 +225,32 @@ var LibraryBrowser = {
       };
       Module['preloadPlugins'].push(audioPlugin);
 
+#if WASM
+      var wasmPlugin = {};
+      wasmPlugin['asyncWasmLoadPromise'] = new Promise((resolve, reject) => resolve());
+      wasmPlugin['canHandle'] = function(name) {
+        return !Module.noWasmDecoding && (name.endsWith('.so') || name.endsWith('.wasm'));
```

Comment:
what is `noWasmDecoding`?

Reply:
I copied the pattern from the audio and image decoders above.  It's a way for users to turn off the decoder if desired, I suppose.  It might make sense to opt-in rather than opt-out of this, but that would be a change from the other two preload plugins.  Happy to change if it makes sense to.